Frontend.Integration.Tests/Pages/Projects/LegalRequirements/DiocesanConsentTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Data.Models;
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Data.TRAMS.ExtensionMethods;
+using Xunit;
+using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+
+namespace Frontend.Integration.Tests.Pages.Projects.LegalRequirements
+{
+    public class DiocesanConsentTests : BaseIntegrationTests
+    {
+        public DiocesanConsentTests(IntegrationTestingWebApplicationFactory factory) : base(factory)
+        {
+        }
+
+        [Fact]
+        public async Task Should_successfully_load_radio_buttons()
```

Comment:
Should there be a test for submitting the form?

Reply:
Done

Reply:
Yeah, sounds good. Added.